.gitignore

```diff
@@ -1,4 +1,4 @@
-
```

Comment:
This is already in there. See the `Node.js` comment in this file

Reply:
Apologies. I did a pull earlier and my program stated that it was up-to-date. I just did another and it is now reflecting the Node.js comment.